plugins/processors/execd/execd.go

```diff
@@ -23,13 +23,20 @@ const sampleConfig = `
 
   ## Delay before the process is restarted after an unexpected termination
   restart_delay = "10s"
+
+  ## Buffer size in bytes for storing metrics received from the process
+  ## The minimum allowed value is 4096
+  buffer_size = 65536
```

Comment:
this is actually the buffer size used for storing a single metric, not multiple metrics.